README.md

```diff
@@ -5,75 +5,46 @@
   Modelina
 </h5>
 <p align="center">
-  <em>The official SDK for generating data models from JSON Schema and AsyncAPI spec</em>
+  <em>The official SDK for generating data models from AsyncAPI and other inputs</em>
 </p>
 
 [![Coverage Status](https://coveralls.io/repos/github/asyncapi/modelina/badge.svg?branch=master)](https://coveralls.io/github/asyncapi/modelina?branch=master)
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-8-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-AsyncAPI Model SDK is a set of classes/functions for generating data models from JSON Schema and AsyncAPI spec.
+Modelina is used to generate different data models, such as Java, TypeScript classes, Go Structs, etc, for AsyncAPI message payloads among other supported inputs.
```

Comment:
Moved to the top instead of https://github.com/asyncapi/modelina/pull/338#discussion_r700663121, too much duplicate text IMO, that explains nothing new 😄